src/DIPS.Xamarin.UI/Controls/Sheet/SheetBehavior.cs

```diff
@@ -551,6 +568,18 @@ private async void ToggleSheetVisibility()
                 return;
             }
 
+            if (!ShouldRememberPosition)
+            {
+                if (m_originalPosition == -1)

```

Comment:
double comparison should always use Math.Abs(val - ( -1)) < EPSILON

Reply:
Sure 👍